peyotl/api/phylesystem_api.py

```diff
@@ -50,7 +50,7 @@ def domain(self):
     @domain.setter
     def domain(self, d):  # pylint: disable=W0221
         self._domain = d
-        self._prefix = '{d}/phylesystem/v1'.format(d=d)
+        self._prefix = '{d}/v3'.format(d=d)
 
```

Comment:
Good catch! i was not aware of this.

Reply:
This was what was causing issues with the updates on production, because while production phylesystempi is still on web2py and is at phylesystemapi.opentreeoflife.org/phylesystem/v1, the peyotl default was pointing to api, and loodking for phylesystemapi at api.opentreeoflife.org/phylesystem/v1, which isn't were it is AND there is no redirect BUT api.opentreeoflife.org/v3 does the redirect properly. Conveniently, this was about to be a problem with the pyramid cahnge as well - but now it should just be handled by api/devapi having the proper re-directs.

Although... I am realizing that we need to make sure the peyotl config points to devapi on devotindex...